### ISSUE TITLE
Excise Tax Reconciliation Analysis

### ISSUE THREAD

**Issue by koushik-ac** [beezlabs-org/automation-dev#52]
We completed the first Clarifications call yesterday with OpEx. We are awaiting some more details before we can close the requirements and initial Analysis.

**Comment by karthikeyan23** [comment]
Needs GR clarity

**Comment by koushik-ac** [comment]
We have decided to first complete the Complexity analysis for BUSA Company Code and get Budget Approval and then only proceed with PDD. The SOP documents provided to us do not define the scope of the automation as they have the total end-to-end process. We are awaiting the clarifications from OpEx contact (Maria Del Mar) to share the scope of the automation and also the input and output files needed by the automation.

We were informed that the recordings of the process which they have are in Spanish language and they have setup a call with end users to go through this process on Monday. Post Monday, we will get an idea as to what is the expected scope of the automation for BUSA and will be able to give correct estimates for BUSA and a rough estimate for all other Company Codes.

**Comment by dominicceo** [comment]
@koushik-ac please put a comment here tagging me so that I will take it with Juan once your analysis is done as Budget approval is key for any effort further on this

**Comment by koushik-ac** [comment]
@dominicceo we have a call setup today with the end users post which I will be able to provide my estimates for this use case.

**Comment by koushik-ac** [comment]
@dominicceo - We have completed the call with business users but still not able to come to a conclusion concerning the scope of the automation. The OpEx contact was not part of the call which caused this issue. I have written an email to all the stakeholders for further action.

This automation is also part of SOX control. We need to check the feasibility of having this included in SOX Automation use case as well.

**Comment by Dean13b** [comment]
Call scheduled today with Juan for budget approval

**Comment by dominicceo** [comment]
@Dean13b Juan is out sick today so mostly this will be done tomorrow

**Comment by koushik-ac** [comment]
The call has been rescheduled to today to discuss about the use case for Budget Approval. Received scope clarifications and needed inputs from the end users.

**Comment by koushik-ac** [comment]
We have scheduled a follow up call with the end users today.

**Comment by mbharathm** [comment]
For further analysis, on RPA Analyst side along with OpEx, a call is being planned with FAS. As this week is MEC, call will be planned next week.
Meanwhile Karla to align with Juan on this approach to prototype our consulting approach.
Bharath to work with Sharon on

**Comment by mbharathm** [comment]
OpEx to get back on the revised requirements as confirmed by Karla. Any change in process steps needs to be taken for analysis and budget approval based on the complexity conclusion.
Following up with Maria from OpEx to get an ETA for the updated requirements.

**Comment by Dean13b** [comment]
Received budget approval

**Comment by Dean13b** [comment]
Dependency with client and a call to be setup for clarification.

**Comment by koushik-ac** [comment]
We have received all the updates from the team. @koushik-ac to start preparing the PDD. Expected to close the PDD by 15-12-2023.

**Comment by dominicceo** [comment]
[like]  Mario Dominic reacted to your message:
________________________________

**Comment by AmitRanjanBasu** [comment]
Review is complete. It is a good quality work

**Comment by koushik-ac** [comment]
Development is progressing as expected. We have a dependency regarding to Screenshot capturing. I plan to discuss on this with Sakthi and Dinesh and close the issue.

**Comment by koushik-ac** [comment]
**Dependencies for this Use Case:**
1. IPA_RTR Access for the transactions ZR151, FAGLB03, MB5B, MB5T, KE24 in Q00 and P00.
2. Dependency with Remote Desktop Connection - For UAT, we can manage this by taking the VM physically.
3. Dependency with Daemon Program Creation and a Separate VM for Screenshot Capturing.
4. Share point location and access to it to upload the final files.

**Comment by mbharathm** [comment]
@koushik-ac ,
* IPA_RTR Access for the transactions ZR151, FAGLB03, MB5B, MB5T, KE24 in Q00 and P00. - Please send the approval email to Chriselda/Security team with details
* I will create business case for new VM and send the details to Juan
* SharePoint location - Please reach out to OpEx/End User

**Comment by mbharathm** [comment]
@koushik-ac , can you share the configuration you need for these remote PCs.

Cc: @Dean13b as well for confirmation.

**Comment by koushik-ac** [comment]
Yes @mbharathm. We would be needing a VM with 8GB Ram. But we are discussing if it will be better if we can ask Juan to get a Physical PC and use it for this purpose. We are waiting for @Kalidass2011 confirmation to take a call. If Kalidass can confirm that this will work with VM also, then we would be needing 8GB Ram configuration VM.

**Comment by mbharathm** [comment]
@koushik-ac , @Kalidass2011 please share the details something like this. Add if something is missing

<img width="521" alt="image" src="https://github.com/beezlabs-org/automation-dev/assets/64949233/3c32e09d-43d7-4130-a473-742ccbf9e38d">

**Comment by koushik-ac** [comment]
Development has been completed. SIT in Progress.

**Comment by koushik-ac** [comment]
We have completed the first UAT Demo. Team will be reviewing the results and get back to us in case of any issues.

**Comment by mbharathm** [comment]
@koushik-ac , as part of the UAT handover milestone, please prepare the test results document from your SIT and share it across all stakeholders for closing the 1st milestone. In the email, include Test Results Document and End User Guide. Copy Priyanka as well in your email. Please send it today.

@priyankag06 once this is sent, we are good to request GR for the 1st milestone.

Cc: @dominicceo

**Comment by koushik-ac** [comment]
Yes Bharath, I shall share them by end of day today.

**Comment by mbharathm** [comment]
@koushik-ac , Juan wanted to understand the technicality behind the screenshot approach on the request made for a new VM. We have to take up this call at the earliest with him to get the procurement done. Lionel team usually takes a week once the request is raised. So we have to close this alignment with Juan for him to take this request to Infra team. Please discuss this with product team and setup a call.

Cc: @dominicceo @swapnamn

**Comment by Dean13b** [comment]
UAT planned to commence from Monday
